src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -1,21 +1,20 @@
 package cromwell.engine.db
 
-import cromwell.engine.WorkflowOutputs
-import wdl4s._
 import cromwell.engine.ExecutionStatus.ExecutionStatus
-import cromwell.engine._
+import cromwell.engine.{WorkflowOutputs, _}
 import cromwell.engine.backend.{Backend, JobKey}
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.{BackendCallKey, ExecutionStoreKey, OutputKey}
 import cromwell.webservice.{CallCachingParameters, WorkflowQueryParameters, WorkflowQueryResponse}
+import wdl4s._
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 object DataAccess {
   val globalDataAccess: DataAccess = new slick.SlickDataAccess()
 }
 
-trait DataAccess {
+trait DataAccess extends AutoCloseable {
```

Comment:
I have not seen other slick code using this, but then again-- I haven't seen _most_ typesafe code (slick, spray, akka, etc.) tidying up. Usually one has to go digging deep into the test code / google groups for finding instructions on how to cleanly shutdown.

What happened here was that I was wiring in the `ExecutionContext`s, when I realized I already made `SlickDataAccess` extend `AutoCloseable` back in PRs #371 / #403. The change was made such that when the liquibase/slick comparison finished, the both databases would be [`.autoClosed`](https://github.com/broadinstitute/cromwell/blob/e95f46dcffe73b0795b7d6b2aa999035500cec97/src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala#L97-L98) using `better.files`' [ARM](https://github.com/pathikrit/better-files#lightweight-arm).